### PR TITLE
updating java cartridges to include the java8 marker and JDK8 path

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by corey112358** [comment]
@danmcp Could you take a look at this when you have time?

File: cartridges/openshift-origin-cartridge-jbossas/bin/util [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 function update-configuration() {
-  if marker_present java7 || [ ${1:-undefined} == java7 ]
+  if marker_present java8 || [ ${1:-undefined} == java8 ]
   then
-    export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK7
-  else
+    export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK8
+  elif marker_present java7 || [ ${1:-undefined} == java7 ]
+  then 
     export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK6
```
Comment (danmcp) [inline cartridges/openshift-origin-cartridge-jbossas/bin/util]: Shouldn't this be JDK7?

**Comment by corey112358** [comment]
@danmcp fixed, thanks for catching that, i'll look over my changes more carefully in the future.

File: cartridges/openshift-origin-cartridge-jbossas/bin/util [hunk 2]
```diff
@@ -3,11 +3,14 @@
...
-  if marker_present java7 || [ ${1:-undefined} == java7 ]
+  if marker_present java8 || [ ${1:-undefined} == java8 ]
   then
+    export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK8
+  elif marker_present java7 || [ ${1:-undefined} == java7 ]
+  then 
     export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK7
   else
-    export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK6
+    export JAVA_HOME=$OPENSHIFT_JBOSSAS_JDK6
```
Comment (danmcp) [inline cartridges/openshift-origin-cartridge-jbossas/bin/util]: Nit: Why add a space here?

**Comment by danmcp** [comment]
@bparees Review please

**Comment by bparees** [comment]
lgtm aside from the nit.

**Comment by bparees** [comment]
[test][extended:cartridge]

**Comment by danmcp** [comment]
[merge]

**Pull request merged**